### ISSUE TITLE
Simple Shuffle Mode Request

### ISSUE THREAD

**Issue by SugarSpikes** [joyrider3774/PlayniteSound#48]
Hi! I love your plugin for playnite. Something I would like to see is the ability to simply shuffle the music, but not be tied to selecting things. Right now, you can have:
1) the music shuffle every selection if you select randomize upon selection
2) the music shuffle but only after it plays through the same first music file in your source folder, if you choose randomize after loop. 

It would be nice to have the option for the first song to always be random, and when it finishes playing, pick another random song to play, and so on, and selecting games/platforms/etc doesn't interfere or reload the songs. Basically a fully random background music player.

Thanks!

**Comment by cnapolit** [comment]
Sounds similar to this issue: https://github.com/joyrider3774/PlayniteSound/issues/24

**Comment by SugarSpikes** [comment]
> Sounds similar to this issue: #24

It's similar, this would be a more generalized and generic version. Music could all be in a singular folder, compared to pulling music from all the individual game-specific music folders.

**Comment by cnapolit** [comment]
Ah, ok. Would this exclusively pull from a single folder, or would you want to be able to select multiple folders to pull from?
I would imagine subfolders would need to be included. I'm also assuming this would not include the game/platform/etc. specific folders.

**Comment by cnapolit** [comment]
Also, what makes this different from having a music player run in the background?
The primary use case for Sounds is playing specific audio files in response to Playnite events.
The only thing I can think of is stopping/starting when a game starts/stops.
However, that can be achieved via scripts in the settings -> scripts section.

**Comment by SugarSpikes** [comment]
Maybe a better way to characterize my issue is this, please ignore what I wrote previously. 

Currently, I have this configuration:
Music Type: Default (all my music is in this folder)
Randomize Music Every Loop: On.

This is what happens:
Playnite launch---->Always plays first alphabetical song in default directory----->Then it plays random music from default directory

What I would like to happen:
Playnite launch----->Plays random music from default directory

That's what I'm interested in.

**Comment by cnapolit** [comment]
Ah ok, that makes sense. Sounds like a bug to me. It should always be random with those settings enabled.

Just attempted to recreate this issue with these settings on v5.6:
![image](https://user-images.githubusercontent.com/94765074/208968474-b703b94f-f917-4d06-9f86-681925892875.png)
it played different music on most launches, which is how it should behave.

Pasting a similar screenshot might help narrow down what's going wrong.

What version of the plugin do you have? It might have been fixed since then.

**Comment by cnapolit** [comment]
Also, sorry for the wait. I've just wrapped up a huge refactor of the plugin (visible in my fork), so I've been rather busy 😅

**Comment by SugarSpikes** [comment]
Hi there! I wanted to follow up here to close to the topic. You helped me out with the issue I was having here on the playnite discord, and the plugin functions as intended. On my end, the problem was having additional, non mp3 files in the same folder. After removing those files and only keeping mp3s in the required folder, everything worked correctly. Thank you!
